# Italian Translation
# Copyright (C) YEAR Free Software Foundation, Inc.
# Daniele Medri <madrid@linux.it>, 2000.
#
msgid ""
msgstr ""
"Project-Id-Version: gnome-db\n"
"POT-Creation-Date: 2001-05-27 14:38+0200\n"
"PO-Revision-Date: 2001-01-14 22:25+01:00\n"
"Last-Translator: Clara Tattoni <clara.tattoni@libero.it>\n"
"Language-Team: Italian <it@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=iso-8859-1\n"
"Content-Transfer-Encoding: 8bit\n"

#: lib/gda-client/gda-connection-pool.c:337
#, c-format
msgid "could not open connection to %s"
msgstr "Impossibile aprire una connessione a %s"

#: lib/gda-client/gda-connection-pool.c:345
#, c-format
msgid "Data source %s not found"
msgstr "Sorgente dati %s non trovata"

#: lib/gda-client/gda-connection.c:235 lib/gda-client/gda-error.c:179
#, c-format
msgid "%s: The server didn't respond."
msgstr "%s: Il server non ha risposto."

#: lib/gda-client/gda-connection.c:239 lib/gda-client/gda-error.c:183
#, c-format
msgid "%s: An Error occured in the CORBA system."
msgstr "%s: Si è verificato un errore nel sistema CORBA."

#: lib/gda-client/gda-connection.c:511
#, c-format
msgid "Database '%s' not found in system configuration"
msgstr "Il database '%s' non è stato trovato nella configurazione del sistema"

#: lib/gda-client/gda-connection.c:526
msgid "Could not open CORBA factory"
msgstr ""

#: lib/gda-client/gda-field.c:245
msgid "<NULL>"
msgstr "<NULL>"

#: lib/gda-client/gda-field.c:251 lib/gda-client/gda-field.c:253
msgid "<Unknown GDA Type(NULL)>"
msgstr "<Tipo GDA sconosciuto (NULL)>"

#: lib/gda-client/gda-field.c:266
msgid "TRUE"
msgstr "SI"

#: lib/gda-client/gda-field.c:267
msgid "FALSE"
msgstr "NO"

#: lib/gda-common/gda-corba.c:75
#, c-format
msgid "CORBA System exception: %s"
msgstr "Eccezione del sistema CORBA: %s"

#: lib/gda-common/gda-corba.c:105
msgid "True"
msgstr "Si"

#: lib/gda-common/gda-corba.c:105
msgid "False"
msgstr "No"

#: lib/gda-common/gda-init.c:59
msgid "Attempt to initialize an already initialized client"
msgstr "Tentativo di inizializzare un client già inizializzato"

#: lib/gda-common/gda-log.c:48
msgid "Saving log...x"
msgstr "Salvataggio del log...x"

#. compose message
#: lib/gda-common/gda-log.c:69
msgid "ERROR: "
msgstr "ERRORE: "

#: lib/gda-common/gda-log.c:69
msgid "MESSAGE: "
msgstr "MESSAGGIO: "

#: lib/gda-common/gda-thread.c:211
msgid "thread is already running"
msgstr "il è thread già in esecuzione"

#: lib/gda-common/gda-xml-database.c:286
#, c-format
msgid "table %s already exists"
msgstr "la tabella %s esiste già"

#: lib/gda-common/gda-xml-database.c:450
#, c-format
msgid "%p is not a valid table"
msgstr "%p non è una tabella valida"

#: lib/gda-server/gda-server-error.c:185
msgid "Could not get pointer to server implementation"
msgstr ""

#: lib/gda-server/gda-server-impl-connection.c:122
#: lib/gda-server/gda-server-impl-connection.c:131
#: lib/gda-server/gda-server-impl-connection.c:138
#: lib/gda-server/gda-server-impl-connection.c:147
#: lib/gda-server/gda-server-impl-connection.c:154
#: lib/gda-server/gda-server-impl-connection.c:163
#: lib/gda-server/gda-server-impl-connection.c:170
#: lib/gda-server/gda-server-impl-connection.c:179
#: lib/gda-server/gda-server-impl-recordset.c:126
#: lib/gda-server/gda-server-impl-recordset.c:135
#: lib/gda-server/gda-server-impl-recordset.c:142
#: lib/gda-server/gda-server-impl-recordset.c:157
#: lib/gda-server/gda-server-impl-recordset.c:166
#: lib/gda-server/gda-server-impl-recordset.c:173
#: lib/gda-server/gda-server-impl-recordset.c:182
#: lib/gda-server/gda-server-impl-recordset.c:189
#: lib/gda-server/gda-server-impl-recordset.c:198
#: lib/gda-server/gda-server-impl-recordset.c:205
#: lib/gda-server/gda-server-impl-recordset.c:213
#: lib/gda-server/gda-server-impl-recordset.c:222
#: lib/gda-server/gda-server-impl-recordset.c:229
#: lib/gda-server/gda-server-impl-recordset.c:237
#: lib/gda-server/gda-server-impl-recordset.c:245
#: lib/gda-server/gda-server-impl-recordset.c:273
#: lib/gda-server/gda-server-impl-recordset.c:281
#: lib/gda-server/gda-server-impl-recordset.c:289
#: lib/gda-server/gda-server-impl-recordset.c:297
#: lib/gda-server/gda-server-impl-recordset.c:305
#: lib/gda-server/gda-server-impl-recordset.c:314
#, c-format
msgid "%s not implemented"
msgstr "%s non implementato"

#: lib/gda-server/gda-server-impl-connection.c:269
#, c-format
msgid "impl_GDA_Connection_open: opening connection with DSN: %s"
msgstr "impl_GDA_Connection_open: apertura di una connessione con DSN: %s"

#: lib/gda-server/gda-server-impl-recordset.c:402
#, c-format
msgid "%s: an error ocurred while fetching data"
msgstr "%s: si è verificato un errore nella raccolta dati"

#: lib/gda-server/gda-server-impl.c:197
#, c-format
msgid "Starting provider %s with no implementation functions"
msgstr "Avvio di un fornitore %s senza funzioni di implementazione"

#: lib/gda-server/gda-server-impl.c:223
#: report/server/gda-report-server-main.c:113
msgid ""
"OAF doesn't know about our IID; indicates broken installation. Exiting..."
msgstr ""
"OAF non conosce questa IID; indice di un'installazione non corretta.  "
"Chiusura in corso... "

#: lib/gda-server/gda-server-impl.c:226
msgid ""
"Another instance of this provider is already registered with OAF. Exiting..."
msgstr ""
"Un'altra istanza di questo provider è già registrata con OAF. Chiusura in "
"corso..."

#: lib/gda-server/gda-server-impl.c:229
msgid "Unknown error registering this provider with OAF. Exiting"
msgstr ""
"Errore sconosciuto nella registrazione di questo provider con OAF. hiusura "
"in corso... "

#: lib/gda-server/gda-server-impl.c:233
#, c-format
msgid "Registered with ID = %s"
msgstr "Registrato con ID = %s"

#: lib/gda-server/gda-server-impl.c:312
#, c-format
msgid "CORBA system exception %s"
msgstr "Eccezione del sistema CORBA %s"

#: lib/gda-server/gda-server-impl.c:315
#, c-format
msgid "CORBA user exception: %s"
msgstr "Eccezione dell'utente CORBA: %s"

#: lib/gda-server/gda-server-init.c:63 lib/gda-server/gda-server-init.c:69
#, c-format
msgid "Received signal %d, shutting down"
msgstr "Ricevuto segnale %d, arresto in corso"

#: lib/gda-server/gda-server-init.c:73
#, c-format
msgid "Received signal %d. Ignoring"
msgstr "Ricevuto il segnale %d, ignorato"

#: lib/gda-server/gda-server-init.c:123
msgid "Attempt to initialize an already initialized provider"
msgstr "Tentativo di inizializzazione di un provider già inizializzato"

#: providers/gda-ldap-server/GNOME_GDA_Provider_LDAP.oaf.in.h:1
msgid "GDA Datasource Access for LDAP"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:273
#: providers/gda-ldap-server/gda-ldap-connection.c:284
#: providers/gda-mdb-server/gda-mdb-connection.c:372
#: providers/gda-mysql-server/gda-mysql-connection.c:370
#: providers/gda-odbc-server/gda-odbc-connection.c:621
#: providers/gda-odbc-server/gda-odbc-connection.c:654
#: providers/gda-oracle-server/gda-oracle-connection.c:438
#: providers/gda-postgres-server/gda-postgres-connection.c:743
#: providers/gda-sybase-server/gda-sybase-error.c:115
#: providers/gda-tds-server/gda-tds-error.c:114
#, c-format
msgid "error '%s' at %s"
msgstr "errore '%s' in %s"

#: providers/gda-interbase-server/gda-interbase-connection.c:278
#: providers/gda-interbase-server/gda-interbase-connection.c:279
#: providers/gda-ldap-server/gda-ldap-connection.c:289
#: providers/gda-ldap-server/gda-ldap-connection.c:290
#: providers/gda-mdb-server/gda-mdb-connection.c:376
#: providers/gda-mdb-server/gda-mdb-connection.c:377
#: providers/gda-mysql-server/gda-mysql-connection.c:374
#: providers/gda-mysql-server/gda-mysql-connection.c:375
#: providers/gda-odbc-server/gda-odbc-connection.c:623
#: providers/gda-odbc-server/gda-odbc-connection.c:624
#: providers/gda-odbc-server/gda-odbc-connection.c:656
#: providers/gda-odbc-server/gda-odbc-connection.c:657
#: providers/gda-oracle-server/gda-oracle-connection.c:443
#: providers/gda-oracle-server/gda-oracle-connection.c:444
#: providers/gda-postgres-server/gda-postgres-connection.c:750
#: providers/gda-postgres-server/gda-postgres-connection.c:751
#: providers/gda-primebase-server/gda-primebase-connection.c:363
#: providers/gda-primebase-server/gda-primebase-connection.c:364
#: providers/gda-sybase-server/gda-sybase-error.c:124
#: providers/gda-sybase-server/gda-sybase-error.c:125
#: providers/gda-tds-server/gda-tds-error.c:123
#: providers/gda-tds-server/gda-tds-error.c:124
msgid "Not available"
msgstr "Non disponibile"

#: providers/gda-interbase-server/gda-interbase-connection.c:280
#: providers/gda-ldap-server/gda-ldap-connection.c:291
#: providers/gda-mdb-server/gda-mdb-connection.c:378
#: providers/gda-mysql-server/gda-mysql-connection.c:376
#: providers/gda-oracle-server/gda-oracle-connection.c:445
#: providers/gda-postgres-server/gda-postgres-connection.c:752
#: providers/gda-primebase-server/gda-primebase-connection.c:365
#: providers/gda-sybase-server/gda-sybase-error.c:126
#: providers/gda-tds-server/gda-tds-error.c:125
msgid "error"
msgstr "errore"

#: providers/gda-ldap-server/main-ldap.c:71
msgid "Could not register LDAP provider implementation"
msgstr "Impossibile registrare l'implementazione del server LDAP"

#: providers/gda-interbase-server/GNOME_GDA_Provider_InterBase.oaf.in.h:1
msgid "GDA Datasource Access for Interbase"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:272
#, c-format
msgid "Error code %ld"
msgstr "Codice di errore %ld"

#: providers/gda-interbase-server/main-interbase.c:72
msgid "Could not register INTERBASE provider implementation"
msgstr "Impossibile registrare l'impelmentazione per il provider INTERBASE"

#: providers/gda-mail-server/GNOME_GDA_Provider_Mail.oaf.in.h:1
msgid "GDA Datasource Access for Mail"
msgstr ""

#: providers/gda-mail-server/gda-mail-connection.c:108
#: providers/gda-mdb-server/gda-mdb-connection.c:224
#: providers/gda-mysql-server/gda-mysql-connection.c:239
#: providers/gda-oracle-server/gda-oracle-connection.c:295
#: providers/gda-postgres-server/gda-postgres-connection.c:477
#: providers/gda-sybase-server/gda-sybase-connection.c:283
#: providers/gda-tds-server/gda-tds-connection.c:288
#, c-format
msgid "Unhandled SCHEMA_QTYPE %d"
msgstr "SCHEMA_QTYPE non gestito %d"

#: providers/gda-mail-server/main-mail.c:72
msgid "Could not register Mail provider implementation"
msgstr "Impossibile registrare l'implementazione per il provider della Posta"

#: providers/gda-mdb-server/GNOME_GDA_Provider_MDB.oaf.in.h:1
msgid "GDA Datasource Access for MDB (MS Access) files"
msgstr ""

#. execute command
#: providers/gda-mdb-server/gda-mdb-command.c:86
#: providers/gda-mysql-server/gda-mysql-command.c:103
#: providers/gda-sybase-server/gda-sybase-command.c:132
#: providers/gda-tds-server/gda-tds-command.c:120
#, c-format
msgid "executing command '%s'"
msgstr "esecuzione del comando '%s'"

#: providers/gda-mdb-server/gda-mdb-command.c:88
#: providers/gda-mysql-server/gda-mysql-command.c:105
#, c-format
msgid "info '%s'"
msgstr "informazioni su '%s'"

#: providers/gda-mdb-server/gda-mdb-connection.c:137
#: providers/gda-mysql-server/gda-mysql-connection.c:140
#, c-format
msgid ""
"%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST or "
"a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT "
"options"
msgstr ""
"%s:%d: impossibile fornire una UNIX_SOCKET se si è già fornito un HOST o una "
"PORTA. Rimuovere la UNIX_SOCKET, oppure rimuovere entrambe le opzioni HOST e "
"PORTA"

#: providers/gda-mdb-server/gda-mdb-connection.c:147
#, c-format
msgid "%s:%d: You must  provide the filename with the MS Access database"
msgstr "%s:%d: bisogna fornire il nome del file con il database di MS Access"

#: providers/gda-mdb-server/gda-mdb-connection.c:152
#, c-format
msgid "Opening connection with filename=%s, user=%s, password=%s"
msgstr "Connessione in corso con file=%s, utente=%s, password=%s"

#: providers/gda-mdb-server/gda-mdb-connection.c:427
#: providers/gda-mysql-server/gda-mysql-connection.c:421
msgid "mysql_recset is NULL"
msgstr "mysql_recset è NULL"

#: providers/gda-mdb-server/gda-mdb-connection.c:469
#: providers/gda-mysql-server/gda-mysql-connection.c:460
#: providers/gda-sybase-server/gda-sybase-connection.c:458
#: providers/gda-tds-server/gda-tds-connection.c:457
msgid "No table name given for SCHEMA_COLS"
msgstr "Nessun nome formito per  SCHEMA_COLS"

#: providers/gda-mdb-server/main-mdb.c:72
#: providers/gda-mysql-server/main-mysql.c:72
msgid "Could not register MySQL provider implementation"
msgstr "Impossibile registrare l'implementazione per il provider MySQL"

#: providers/gda-mysql-server/GNOME_GDA_Provider_MySQL.oaf.in.h:1
msgid "GDA Datasource Access for MySQL"
msgstr ""

#: providers/gda-mysql-server/gda-mysql-connection.c:153
#, c-format
msgid ""
"Opening connection with user=%s, password=%s, host=%s, port=%s, unix_socket=%"
"s"
msgstr ""
"Apertura di una connessione per l'utente=%s, password=%s, host=%s, porta=%s, "
"unix_socket=%s"

#: providers/gda-odbc-server/GNOME_GDA_Provider_ODBC.oaf.in.h:1
msgid "GDA Datasource Access for ODBC"
msgstr ""

#: providers/gda-odbc-server/gda-odbc-connection.c:280
#, c-format
msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
msgstr "gda_odbc_connection_open_schema: SCHEMA_QTYPE %d non gestito\n"

#: providers/gda-odbc-server/gda-odbc-connection.c:638
#: providers/gda-odbc-server/gda-odbc-connection.c:639
#: providers/gda-odbc-server/gda-odbc-connection.c:640
#: providers/gda-odbc-server/gda-odbc-connection.c:671
#: providers/gda-odbc-server/gda-odbc-connection.c:672
#: providers/gda-odbc-server/gda-odbc-connection.c:673
msgid "no text"
msgstr "testo assente"

#: providers/gda-odbc-server/main-odbc.c:71
msgid "Could not register ODBC provider implementation"
msgstr "Impossibile registrare l'implementazione per il provider ODBC"

#: providers/gda-oracle-server/GNOME_GDA_Provider_Oracle.oaf.in.h:1
msgid "GDA Datasource Access for Oracle"
msgstr ""

#: providers/gda-oracle-server/gda-oracle-command.c:185
msgid "Could not allocate statement handle"
msgstr ""

#: providers/gda-oracle-server/gda-oracle-command.c:189
msgid "command object being created without associated connection"
msgstr "il comando è stato creato senza una connessione associata"

#: providers/gda-oracle-server/gda-oracle-command.c:220
#, c-format
msgid "Executing command '%s'"
msgstr "Esecuzione del comando '%s'"

#: providers/gda-oracle-server/gda-oracle-connection.c:113
msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
msgstr "errore in OCIHandleAlloc() HTYPE_SERVER"

#: providers/gda-oracle-server/gda-oracle-connection.c:116
msgid "error in OCIHandleAlloc() HTYPE_SERVER"
msgstr "errore in OCIHandleAlloc() HTYPE_SERVER"

#: providers/gda-oracle-server/gda-oracle-connection.c:119
msgid "error in OCIHandleAlloc() HTYPE_ERROR"
msgstr "errore in OCIHandleAlloc() HTYPE_ERROR"

#: providers/gda-oracle-server/gda-oracle-connection.c:379
#, c-format
msgid "Unknown ORACLE type %ld"
msgstr "Tipo ORACLE sconosciuto %ld"

#: providers/gda-oracle-server/gda-oracle-recordset.c:115
msgid "Unknown ORACLE data type in fill_field_values"
msgstr "Tipo di dato ORACLE sconosciuto in fill_field_values"

#: providers/gda-oracle-server/gda-oracle-recordset.c:150
msgid "recordset object being created without associated connection"
msgstr ""

#: providers/gda-oracle-server/main-oracle.c:71
msgid "Could not register ORACLE provider implementation"
msgstr "Impossibile registrare l'implementazione per il provider ORACLE"

#: providers/gda-postgres-server/GNOME_GDA_Provider_Postgres.oaf.in.h:1
msgid "GDA Datasource Access for Postgres"
msgstr ""

#: providers/gda-postgres-server/gda-postgres-connection.c:849
#: providers/gda-postgres-server/gda-postgres-connection.c:957
msgid "postgres_recset is NULL"
msgstr "postgres_recset è NULL"

#: providers/gda-postgres-server/main-postgres.c:71
msgid "Could not register PostgreSQL provider implementation"
msgstr "Impossibile registrare l'implementazione per il provider PostgreSQL"

#: providers/gda-sybase-server/GNOME_GDA_Provider_Sybase.oaf.in.h:1
msgid "GDA Datasource Access for Sybase"
msgstr ""

#.      gda_server_error_make(error, 0, cnc, "connection seems to be dead");
#: providers/gda-sybase-server/gda-sybase-command.c:127
#: providers/gda-tds-server/gda-tds-command.c:116
msgid "Connection seems to be dead"
msgstr "La connessione sembra essere morta"

#: providers/gda-sybase-server/gda-sybase-command.c:146
#: providers/gda-tds-server/gda-tds-command.c:131
msgid "Filling command structure failed\n"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-command.c:158
#: providers/gda-tds-server/gda-tds-command.c:139
msgid "sending cmd failed"
msgstr "invio del comando fallito"

#: providers/gda-sybase-server/gda-sybase-command.c:165
#: providers/gda-tds-server/gda-tds-command.c:146
msgid "Allocating recordset failed"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-command.c:175
#: providers/gda-tds-server/gda-tds-command.c:154
msgid "Getting recordset userdata failed"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-command.c:233
#: providers/gda-tds-server/gda-tds-command.c:207
msgid "Command execution finished, all results processed"
msgstr ""
"Esecuzione del comando terminata, tutti i risultati sonon stati processati"

#: providers/gda-sybase-server/gda-sybase-command.c:237
#: providers/gda-tds-server/gda-tds-command.c:211
msgid "Command failed"
msgstr "Comando fallito"

#: providers/gda-sybase-server/gda-sybase-command.c:241
#: providers/gda-tds-server/gda-tds-command.c:215
msgid "Execution of resultless command succeeded"
msgstr "Comandi senza risultati eseguiti con successo"

#: providers/gda-sybase-server/gda-sybase-command.c:257
#: providers/gda-tds-server/gda-tds-command.c:231
#, c-format
msgid "ct_results() returned %x"
msgstr "ct_results() fornisce %x"

#: providers/gda-sybase-server/gda-sybase-command.c:267
#: providers/gda-tds-server/gda-tds-command.c:241
msgid "No further results, command completly succeeded."
msgstr "Non ci sono ulteriori risultati, comando completato con successo."

#: providers/gda-sybase-server/gda-sybase-command.c:271
#: providers/gda-tds-server/gda-tds-command.c:245
msgid "An error occured during command execution."
msgstr "Si è verificato un errore durante l'esecuzione del comando."

#: providers/gda-sybase-server/gda-sybase-command.c:278
#: providers/gda-tds-server/gda-tds-command.c:249
msgid "Cmd execution returned unknown result, may be an error"
msgstr ""
"L'esecuzione del comando ha fornito un risultato sconosciuto, può essere un "
"errore"

#: providers/gda-sybase-server/gda-sybase-command.c:291
#: providers/gda-tds-server/gda-tds-command.c:259
#, fuzzy
msgid "No data returned, freeing recordset structure"
msgstr "Nessun dato fornito, "

#: providers/gda-sybase-server/gda-sybase-command.c:322
#: providers/gda-sybase-server/gda-sybase-command.c:334
#: providers/gda-tds-server/gda-tds-command.c:289
#: providers/gda-tds-server/gda-tds-command.c:297
msgid "Freeing command structure failed"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-connection.c:171
msgid "Not enough memory for sybase_Connection"
msgstr "Memoria insufficiente per sybase_Connection"

#: providers/gda-sybase-server/gda-sybase-connection.c:448
#: providers/gda-tds-server/gda-tds-connection.c:447
#, c-format
msgid "schema_cols: table_name '%s'"
msgstr "schema_cols: table_name '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:464
#: providers/gda-tds-server/gda-tds-connection.c:463
msgid "Could not allocate command structure"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-connection.c:486
#: providers/gda-tds-server/gda-tds-connection.c:485
msgid "Could not allocate enough memory for query"
msgstr "Impossibile allocare abbastanza memoria per la query"

#. 		gda_server_error_make(error, NULL, cnc, __PRETTY_FUNCTION__);
#: providers/gda-sybase-server/gda-sybase-connection.c:500
#: providers/gda-tds-server/gda-tds-connection.c:499
msgid "Could not get recordset in schema_cols"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-connection.c:549
#: providers/gda-tds-server/gda-tds-connection.c:548
#, c-format
msgid "schema_procs: invalid constraint type %d"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-connection.c:559
#: providers/gda-tds-server/gda-tds-connection.c:558
msgid "Could not allocate cmd structure"
msgstr "Non è possibile allocare la struttura comandi"

#: providers/gda-sybase-server/gda-sybase-connection.c:626
#: providers/gda-tds-server/gda-tds-connection.c:625
#, c-format
msgid "schema_tables: invalid constraint type %d"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-connection.c:687
#: providers/gda-tds-server/gda-tds-connection.c:686
#, c-format
msgid "Invalid or unsupported constraint type: %d"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-connection.c:728
#: providers/gda-tds-server/gda-tds-connection.c:727
msgid "Could not allocate GdaServerCommand"
msgstr "Impossiobile allocare GdaServerCommand"

#: providers/gda-sybase-server/gda-sybase-connection.c:758
#: providers/gda-sybase-server/gda-sybase-connection.c:763
#: providers/gda-tds-server/gda-tds-connection.c:757
#: providers/gda-tds-server/gda-tds-connection.c:762
#, c-format
msgid "connection status: %x"
msgstr "stato della connessione: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:759
#: providers/gda-tds-server/gda-tds-connection.c:758
#, c-format
msgid "return code: %x"
msgstr "codice di ritorno: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:796
#: providers/gda-tds-server/gda-tds-connection.c:793
msgid "attempting reconnection"
msgstr "tentativo di riconnettersi"

#: providers/gda-sybase-server/gda-sybase-connection.c:935
#, c-format
msgid "Reconnected to '%s'"
msgstr "Riconnesso a %s"

#: providers/gda-sybase-server/gda-sybase-connection.c:937
#: providers/gda-tds-server/gda-tds-connection.c:898
#, c-format
msgid "Connected to %s"
msgstr "Connesso a %s"

#: providers/gda-sybase-server/gda-sybase-connection.c:945
#: providers/gda-tds-server/gda-tds-connection.c:908
#, c-format
msgid "Selected database '%s'"
msgstr "Database selezionato '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:974
#: providers/gda-tds-server/gda-tds-connection.c:933
#, c-format
msgid "%s: Could not allocate cmd structure"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-connection.c:983
#: providers/gda-tds-server/gda-tds-connection.c:942
#, c-format
msgid "%s: Could not allocate memory for cmd string"
msgstr "%s: non è possibile allocare memoria per la linea di comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:996
#: providers/gda-tds-server/gda-tds-connection.c:952
#, c-format
msgid "%s: ct_command failed"
msgstr "%s: comando_ct fallito"

#: providers/gda-sybase-server/gda-sybase-connection.c:1005
#: providers/gda-tds-server/gda-tds-connection.c:961
#, c-format
msgid "%s: sending command failed"
msgstr "%s: invio del comando fallito"

#: providers/gda-sybase-server/gda-sybase-connection.c:1016
#, c-format
msgid "%s: selecting database failed"
msgstr "%s: selezione di un database fallito"

#: providers/gda-sybase-server/gda-sybase-connection.c:1034
#, c-format
msgid "%s: use command failed"
msgstr "%s: uso del comando fallito"

#: providers/gda-sybase-server/gda-sybase-connection.c:1061
#: providers/gda-tds-server/gda-tds-connection.c:1023
msgid "Could not allocate locale structure"
msgstr "Non è possibile allocare la struttura locale"

#: providers/gda-sybase-server/gda-sybase-connection.c:1067
#: providers/gda-tds-server/gda-tds-connection.c:1029
#, c-format
msgid "Could not set locale to '%s'"
msgstr "Non è possibile impostare locale a '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1073
#: providers/gda-tds-server/gda-tds-connection.c:1035
#, c-format
msgid "Could not configure locale to be '%s'"
msgstr "Non è possibile configurare locale a '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1078
#: providers/gda-tds-server/gda-tds-connection.c:1040
msgid "Could not drop locale structure"
msgstr "Non è possibile ignorare la struttura locale"

#: providers/gda-sybase-server/gda-sybase-connection.c:1091
#: providers/gda-tds-server/gda-tds-connection.c:1053
#, c-format
msgid "Fatal error: %s\n"
msgstr "Errore fatale: %s\n"

#: providers/gda-sybase-server/gda-sybase-error.c:71
#, c-format
msgid "gda_sybase_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-sybase-server/gda-sybase-error.c:80
#: providers/gda-tds-server/gda-tds-error.c:78
#, c-format
msgid "No error flag ist set, though %s is called. This may be a bug."
msgstr ""

#: providers/gda-sybase-server/gda-sybase-error.c:144
msgid "cs_diag() initialized"
msgstr "cs_diag() initializzato"

#: providers/gda-sybase-server/gda-sybase-error.c:147
msgid "Could not initialize cs_diag()"
msgstr "Non è possibile inizializzare cs_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:154
msgid "ct_diag() initialized"
msgstr "ct_diag() initializzato"

#: providers/gda-sybase-server/gda-sybase-error.c:157
msgid "Could not initialize ct_diag()"
msgstr "Impossibile inizializzare ct_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:191
#, c-format
msgid "%s could not find server implementation"
msgstr "%s impossibile trovare l'implementazione del server"

#: providers/gda-sybase-server/gda-sybase-error.c:197
#, c-format
msgid "%s could not get cnc userdata"
msgstr "%s impossibile avere cnc userdata"

#: providers/gda-sybase-server/gda-sybase-error.c:209
msgid "Could not get # of messages"
msgstr "Impossibile avere il numero dei messaggi"

#: providers/gda-sybase-server/gda-sybase-error.c:214
#, c-format
msgid "%d ctlib messages to fetch"
msgstr "%d messaggi di ctlib da scaricare"

#: providers/gda-sybase-server/gda-sybase-error.c:216
#, c-format
msgid "%d ctlib message to fetch"
msgstr "%d messaggio di ctlib da scaricare"

#: providers/gda-sybase-server/gda-sybase-error.c:252
msgid "error is nullpointer in gda_sybase_messages_handle"
msgstr "l'errore è un puntatore nullo in gda_sybase_messages_handle"

#: providers/gda-sybase-server/gda-sybase-error.c:277
#: providers/gda-sybase-server/gda-sybase-error.c:344
#: providers/gda-sybase-server/gda-sybase-error.c:387
msgid "Could not get # of server messages"
msgstr "Impossibile avere il numero di messaggi del server"

#: providers/gda-sybase-server/gda-sybase-error.c:285
#, c-format
msgid "Fetching %d client messages"
msgstr "Scaricamento %d messaggi del client"

#: providers/gda-sybase-server/gda-sybase-error.c:287
#, c-format
msgid "Fetching %d client message"
msgstr "Scaricamento %d messaggio del client"

#: providers/gda-sybase-server/gda-sybase-error.c:296
#, c-format
msgid "Could not get client msg # %d"
msgstr "Non è possibile avere il client msg # %d"

#: providers/gda-sybase-server/gda-sybase-error.c:312
msgid "Could not get # of client messages"
msgstr "Impossibile avere il numero dei messaggi del client"

#: providers/gda-sybase-server/gda-sybase-error.c:320
msgid "Could not clear client messages"
msgstr "Impossibile azzerare i messaggi del client"

#: providers/gda-sybase-server/gda-sybase-error.c:352
#, c-format
msgid "Fetching %d server messages"
msgstr "Scaricamento %d messaggi del server"

#: providers/gda-sybase-server/gda-sybase-error.c:354
#, c-format
msgid "Fetching %d server message"
msgstr "Scaricamento %d messaggio del server"

#: providers/gda-sybase-server/gda-sybase-error.c:363
#, c-format
msgid "Could not get server msg # %d"
msgstr "Impossibile prendere il messaggio del server numero %d"

#: providers/gda-sybase-server/gda-sybase-error.c:395
msgid "Could not clear server messages"
msgstr "Impossibile azzerare i messaggi del server"

#: providers/gda-sybase-server/gda-sybase-error.c:417
#: providers/gda-sybase-server/gda-sybase-error.c:453
msgid "Could not get # of cslib messages"
msgstr "Impossibile avere il numero dei messaggi di cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:425
#, c-format
msgid "Fetching %d cslib messages"
msgstr "Scaricamento %d messaggi di cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:427
#, c-format
msgid "Fetching %d cslib message"
msgstr "Scaricamento %d messaggo di cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:437
#, c-format
msgid "Could not get cslib msg # %d"
msgstr "Impossibile avere il  messaggio numero %d di cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:461
msgid "Could not clear cslib messages"
msgstr "Impossibile azzerare i messaggi di cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:593
#: providers/gda-tds-server/gda-tds-error.c:257
msgid "Server message:"
msgstr "Messaggio del Server:"

#: providers/gda-sybase-server/gda-sybase-error.c:602
#: providers/gda-tds-server/gda-tds-error.c:266
msgid "Client message:"
msgstr "Messaggio del Client:"

#: providers/gda-sybase-server/gda-sybase-error.c:609
#: providers/gda-tds-server/gda-tds-error.c:273
msgid "CS-Library error:"
msgstr "Errore di CS-Library:"

#: providers/gda-sybase-server/gda-sybase-recordset.c:158
#: providers/gda-tds-server/gda-tds-recordset.c:155
#, c-format
msgid "fetch error at row %d"
msgstr "trovato un errore alla riga %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:165
#: providers/gda-tds-server/gda-tds-recordset.c:162
#, c-format
msgid "CS_END_DATA on row %d; canceling"
msgstr "CS_END_DATA alla riga %d; annullo in corso"

#: providers/gda-sybase-server/gda-sybase-recordset.c:174
#: providers/gda-tds-server/gda-tds-recordset.c:169
#, c-format
msgid "%s: Error (%d)"
msgstr "%s Errore: (%d)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:349
#: providers/gda-tds-server/gda-tds-recordset.c:342
msgid "Failed fetching # of columns"
msgstr "Tentativo di trovare il numero di colonne fallito"

#: providers/gda-sybase-server/gda-sybase-recordset.c:355
#: providers/gda-tds-server/gda-tds-recordset.c:349
#, c-format
msgid "Counted %d columns"
msgstr "%d colonne contate"

#: providers/gda-sybase-server/gda-sybase-recordset.c:362
#: providers/gda-tds-server/gda-tds-recordset.c:356
#, c-format
msgid "%s could not allocate datafmt info"
msgstr "%s non è possibile allocare le informazioni datafmt"

#: providers/gda-sybase-server/gda-sybase-recordset.c:372
#: providers/gda-tds-server/gda-tds-recordset.c:366
#, c-format
msgid "%s could not allocate data fields"
msgstr "%s non è possibile allocare i campi dei dati"

#: providers/gda-sybase-server/gda-sybase-recordset.c:390
#: providers/gda-tds-server/gda-tds-recordset.c:384
#, c-format
msgid "ct_describe on col # %d"
msgstr "ct_describe sulla col num %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:397
#: providers/gda-tds-server/gda-tds-recordset.c:391
#, c-format
msgid "ct_describe on col # %d failed"
msgstr "ct_describe sulla col num %d fallito"

#: providers/gda-sybase-server/gda-sybase-recordset.c:403
#: providers/gda-tds-server/gda-tds-recordset.c:397
#, c-format
msgid "ct_describe(%d): '%s', %d (%s)"
msgstr "\"ct_describe(%d): '%s', %d (%s)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:412
#: providers/gda-tds-server/gda-tds-recordset.c:406
#, fuzzy
msgid "could not allocate data holder"
msgstr "Impossibile allocare il contenitore dei dati"

#: providers/gda-sybase-server/gda-sybase-recordset.c:422
#: providers/gda-tds-server/gda-tds-recordset.c:416
msgid "could not ct_bind data holder to column"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-recordset.c:429
#: providers/gda-tds-server/gda-tds-recordset.c:423
msgid "illegal type detected, aborting"
msgstr "è stato riconosciuto un dato di tipo illegale, interruzione"

#: providers/gda-sybase-server/gda-sybase-recordset.c:435
#: providers/gda-tds-server/gda-tds-recordset.c:429
msgid "could not allocate field"
msgstr "Non è possibile allocare il campo"

#: providers/gda-sybase-server/main-sybase.c:76
msgid "Could not register gda-sybase provider implementation"
msgstr "Impossibile registrare l'implementazione di gda-sysbase provider"

#: providers/gda-primebase-server/GNOME_GDA_Provider_Primebase.oaf.in.h:1
msgid "GDA Datasource Access for primebase"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:158
#, fuzzy, c-format
msgid "Could not connect to '%s@%s:%s' with connparm '%s'"
msgstr "Impossibile aprire una connessione a %s"

#: providers/gda-primebase-server/gda-primebase-connection.c:162
#, c-format
msgid "%2ld: Connected to '%s@%s:%s' with connparm: '%s'"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:168
msgid "Could not get PBIConnectionInfo(), disconnecting"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:172
#, c-format
msgid "%2ld: ConInfo: '%s@%s', Cl v%x, Srv '%s' v%x, Brand '%s'"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:376
#, fuzzy
msgid "Not enough memory for primebase_Error structure."
msgstr "Memoria insufficiente per sybase_Connection"

#: providers/gda-primebase-server/gda-primebase-connection.c:383
#, c-format
msgid "%2ld: ERROR: %ld (%ld) : %s.\n"
msgstr ""

#: providers/gda-primebase-server/main-primebase.c:70
#, fuzzy
msgid "Could not register PRIMEBASE provider implementation"
msgstr "Impossibile registrare l'impelmentazione per il provider INTERBASE"

#: providers/gda-tds-server/GNOME_GDA_Provider_TDS.oaf.in.h:1
msgid "GDA Datasource Access for Sybase/MS SQL"
msgstr ""

#: providers/gda-tds-server/gda-tds-connection.c:167
#, fuzzy
msgid "Not enough memory for tds_Connection"
msgstr "Memoria insufficiente per sybase_Connection"

#. Fixme: replace "dummy" with buf when CS_GET CS_SERVERNAME works
#: providers/gda-tds-server/gda-tds-connection.c:896
#, fuzzy, c-format
msgid "Reconnected to %s"
msgstr "Riconnesso a %s"

#: providers/gda-tds-server/gda-tds-error.c:69
#, fuzzy, c-format
msgid "gda_tds_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-tds-server/main-tds.c:76
#, fuzzy
msgid "Could not register gda-tds provider implementation"
msgstr "Impossibile registrare l'implementazione di gda-sysbase provider"

#: report/gda-report-engine.c:209
#, c-format
msgid "CORBA exception unloading report engine: %s"
msgstr ""

#: report/server/GNOME_GDA_Report.oaf.in.h:1
msgid "GDA Distributed Report Engine"
msgstr ""

#: report/server/gda-report-server-main.c:41
#: report/server/gda-report-server-main.c:48
#, c-format
msgid "Received signal %d, shutting down."
msgstr "Ricevuto il segnale %d, arresto in corso."

#: report/server/gda-report-server-main.c:52
#, fuzzy, c-format
msgid "Received signal %d, shutting down cleanly"
msgstr "Ricevuto segnale %d, arresto in corso"

#: report/server/gda-report-server-main.c:92
msgid "Failed to initialize OAF: please mail bug report to OAF maintainers"
msgstr ""
"Inizializzazione di OAF non riuscita: inviare un bug report ai mantenitori "
"di OAF. "

#: report/server/gda-report-server-main.c:101
msgid "Could not activate report engine"
msgstr ""

#: report/server/gda-report-server-main.c:116
msgid ""
"Another instance of the report engine is already registered with OAF. "
"Exiting..."
msgstr ""

#: report/server/gda-report-server-main.c:119
msgid "Unknown error registering report engine with OAF. Exiting"
msgstr ""

#: report/server/gda-report-server-main.c:123
#, c-format
msgid "Registered report engine with ID = %s"
msgstr ""

#: tools/gda-run.c:109
#, c-format
msgid "%s: misconfigured DSN entry"
msgstr "%s: voce DNS malconfigurata "

#: tools/gda-run.c:124
#, c-format
msgid "%s: data source '%s' not found\n"
msgstr "%s: sorgente dati '%s' non trovata\n"

#~ msgid "%s: length = %d"
#~ msgstr "%s: lunghezza = %d"

#~ msgid "%s: sending field description for '%s'"
#~ msgstr "%s: invio della descrizione del campo per '%s'"

#~ msgid "schema_tables: table name = '%s'"
#~ msgstr "schema_tables: table name = '%s'"

#~ msgid "schema_tables: table_schema = '%s'"
#~ msgstr "schema_tables: table_schema = '%s'"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!\n"
#~ msgstr "schema_procedures: proc_catalog = '%s' NON USATO!\n"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!"
#~ msgstr "schema_procedures: proc_catalog = '%s' NON USATO!"

#~ msgid "database name"
#~ msgstr "nome del database"

#~ msgid "DATABASE"
#~ msgstr "DATABASE"

#~ msgid "user name"
#~ msgstr "user name"

#~ msgid "USER-NAME"
#~ msgstr "USER-NAME"

#~ msgid "password"
#~ msgstr "password"

#~ msgid "PASSWORD"
#~ msgstr "PASSWORD"

#~ msgid "file name"
#~ msgstr "nome file"

#~ msgid "FILE-NAME"
#~ msgstr "NOME-FILE"
